installer.py

```diff
@@ -143,25 +149,51 @@ def _record_install(dep):
     def bin_dir(*x):
         return osp.join(args.bin_dir, *x)
 
-    has_npm_install = which("npm") or osp.isfile(bin_dir("npm"))
-    # ^ `npm` because `node` is `nodejs` on Ubuntu. `npm` is consistent.
-    # Disjunction b/c `npm` might be in ~/.local/bin but not on PATH.
-    if not args.no_node and (args.force or not has_npm_install):
+    def get_node_version():
+        # ^ `node` is `nodejs` on Ubuntu.
```

Comment:
Are you sure you don't have `node`? For me, `node` works fine on Ubuntu. I do also have `nodejs`, which is a REPL.

Reply:
Node.js really is `nodejs` on Ubuntu. The actual `node` package is some proxy thing or something.

```suggestion
        """Returns the un-prefixed semver of the Node.js executable."""
```
also, if you care to do so, you could also check to see whether [nvm](https://github.com/nvm-sh/nvm) is installed and use that. But in that case, the hardcore js dev would specify `--no-node`, so it's nbd.

Reply:
@mitjat I checked and it's both `node` and `nodejs` on 18.04 and later. On 16.04, it's just `nodejs`.

Reply:
@nhynes I don't have `nvm` on my system, nor on the base docker image I use for testing. We depend on `node` anyway, so I would only stick to that.